src/types/schema-types.ts

```diff
@@ -0,0 +1,234 @@
+export type Maybe<T> = T | null;
```

Comment:
Copying `schema-types.ts` into `src` folder solves issue with restricted imports outside `/src` folder like enums (types are allowed to import). However now we need to maintain two versions of types right now...

1) `server/src/generated/schema-types.ts`
2) `src/types/schema-types.ts`

I see two problems that we need to sort out: 🤔

1) Currently most existing components import types from `server/src/generated/schema-types.ts` (not from `/src/..`) which was fine, because when schema will be updated in the future the components will follow this change... but to remain consistent - from now - I guess we should import all types from `/src/types/schema-types.ts`, right?  
That's being said, we also should update existing imports. 

2) If `server/src/generated/schema-types.ts` will be updated at some point (or to precisely generated again) this change should also be reflected inside `src/types/schema-types.ts`.  I guess it's not _a big_ problem since we probably won't make such update very often... but still it's important to have the **same type definitions shared between backend nad frontend** and having two different files is not making it easy.

Reply:
Very good point @michaelmoney  👍 So:
1. I'm gonna change all possible imports for frontend part to use types from `/src/types/schema-types.ts`
2. `server/src/generated/schema-types` it's a bit different - it includes resolver types as well, which we don't need for frontend. Still any update of the current schema includes an updates for both `server/src/generated/schema-types.ts` and `src/types/schema-types.ts`. In the future, when real backend will be in the progress we can have budy app schema types as a separate `node_module` to have it in one place both for server and frontend projects. Just load it as ```"@types/buddy-app-schema": "0.0.1"``` in `package.json`

Reply:
I agree with @michaelmoney that we should have only one source for types, so we avoid this problem that the front end types are not the same as backend. But not like now that we 2 souces for the types.